nixos/lib/test-driver/test-driver.py

```diff
@@ -462,9 +462,10 @@ def shell_interact(self) -> None:
         Should only be used during test development, not in the production test."""
         self.connect()
         self.log("Terminal is ready (there is no prompt):")
```

Comment:
Is this still relevant?

Reply:
Yes. socat has a prompt option but it does not work for unix sockets.